actions/hybrid_branch_deploy/action.yml

```diff
@@ -71,15 +71,18 @@ runs:
       id: deploy
       with:
         organization_id: ${{ inputs.organization_id }}
-        pr: "${{ github.event.number }}"
-        pr_status: "${{ github.event.pull_request.merged && 'merged' || github.event.pull_request.state }}"
+        pr: "${{ github.event.number || github.event.inputs.dbtref }}"
+        pr_status: "${{ github.event.pull_request.merged && 'merged' || github.event.pull_request.state || 'open' }}"
```

Comment:
why the addition of 'open' here? wouldn't we only want this to happen on a merge?

Reply:
that's the beautiful thing about [copy-paste](https://github.com/slopp/dagsterproj/blob/main/.github/workflows/branch_deployments.yml#L112) - you get to do terrible things like write code without an explanation 😂  but more seriously, i think this evaluates to null when triggered from the dbt project and since it's a branch deployment, we can assume the status of the dbt PR that kicked it off is `open`

Reply:
you're probably thinking of prod deployments, which would happen when the change is merged